t/op/hash.t

```diff
@@ -100,13 +100,12 @@ sub guard::DESTROY {
 }
 
 # Weak references to pad hashes
-SKIP: {
-    skip_if_miniperl("No Scalar::Util::weaken under miniperl", 1);
+{
```

Comment:
These outer braces are also no longer needed

Reply:
Perhaps not, but I'll keep them in because they still guard the `$ref` lexical later on,